api/src/main/java/org/xwiki/filemanager/internal/reference/DefaultUniqueDocumentReferenceGenerator.java

```diff
@@ -86,20 +86,21 @@ public class DefaultUniqueDocumentReferenceGenerator
      */
     @Inject
     private ComponentManager componentManager;
-    
+
     /**
      * Transform a name according to the current name strategy, if the configuration is set to use transformation.
-     * Else it will just return the given name.
-     *
-     * Taken from org.xwiki.model.validation.script.ModelValidationScriptService
+     * Otherwise, it will just return the given name.
+     * <p>
```

Comment:
Why is the `<p>` needed?

Reply:
It was inserted by the codestyle formatter to keep the description above separate from the 'Taken from' specified below. It is not needed necessarily, I can remove it.

Reply:
Ok, I understand. It's ok to keep it:)